docs/source/data/data_catalog.md

```diff
@@ -720,125 +131,45 @@ cars:
-        ),
-        "scooters_query": SQLQueryDataSet(
-            sql="select * from cars where gear=4",
-            credentials=dict(con="sqlite:///kedro.db"),
-        ),
-        "ranked": ParquetDataSet(filepath="ranked.parquet"),
-    }
-)
-```
-
-When using `SQLTableDataSet` or `SQLQueryDataSet` you must provide a `con` key containing [SQLAlchemy compatible](https://docs.sqlalchemy.org/en/13/core/engines.html#database-urls) database connection string. In the example above we pass it as part of `credentials` argument. Alternative to `credentials` is to put `con` into `load_args` and `save_args` (`SQLTableDataSet` only).
-
-### Load datasets
-
-You can access each dataset by its name.
-
-```python
-cars = io.load("cars")  # data is now loaded as a DataFrame in 'cars'
-gear = cars["gear"].values
-```
-
-#### Behind the scenes
-
-The following steps happened behind the scenes when `load` was called:
-
-- The value `cars` was located in the Data Catalog
-- The corresponding `AbstractDataset` object was retrieved
-- The `load` method of this dataset was called
-- This `load` method delegated the loading to the underlying pandas `read_csv` function
-
-### View the available data sources
+A dataset offers versioning support if it extends the [`AbstractVersionedDataSet`](/kedro.io.AbstractVersionedDataSet) class to accept a version keyword argument as part of the constructor and adapt the `_save` and `_load` method to use the versioned data path obtained from `_get_save_path` and `_get_load_path` respectively. In Kedro version 0.18.2, the following datasets support versioning:
+
+- `kedro_datasets.api.APIDataSet`
+- `kedro_datasets.holoviews.HoloviewsWriter`
+- `kedro_datasets.json.JSONDataSet`
+- `kedro_datasets.matplotlib.MatplotlibWriter`
+- `kedro_datasets.networkx.NetworkXDataSet`
+- `kedro_datasets.pandas.CSVDataSet`
+- `kedro_datasets.pandas.ExcelDataSet`
+- `kedro_datasets.pandas.FeatherDataSet`
+- `kedro_datasets.pandas.HDFDataSet`
+- `kedro_datasets.pandas.JSONDataSet`
+- `kedro_datasets.pandas.ParquetDataSet`
+- `kedro_datasets.pickle.PickleDataSet`
+- `kedro_datasets.pillow.ImageDataSet`
+- `kedro_datasets.tensorflow.TensorFlowModelDataSet`
+- `kedro_datasets.text.TextDataSet`
+- `kedro_datasets.spark.SparkDataSet`
+- `kedro_datasets.yaml.YAMLDataSet`
 
-If you forget what data was assigned, you can always review the `DataCatalog`.
-
-```python
-io.list()
-```
-
-### Save data
-
-You can save data using an API similar to that used to load data.
-
-```{warning}
-This use is not recommended unless you are prototyping in notebooks.
-```
-
-#### Save data to memory
-
-```python
-from kedro.io import MemoryDataSet
-
-memory = MemoryDataSet(data=None)
-io.add("cars_cache", memory)
-io.save("cars_cache", "Memory can store anything.")
-io.load("cars_cache")
+```{note}
+Although HTTP(S) is a supported file system in the dataset implementations, it does not support versioning.
```

Comment:
This reads to me like the HTTP(S) file system does not support versioning, but isn't it more like "if you use HTTP(S) as a location, you can't use versioning"?

(By the way does this give an error or anything?)